spec.md

```diff
@@ -304,6 +311,28 @@ Location: <blob-location>
 Here, `<blob-location>` is a pullable blob URL.
 
 
+##### Mounting a blob from another repository
+If a necessary blob exists already in another repository, it can be mounted into a different repository via a `POST`
+request in the following format:
+
+`/v2/<name>/blobs/uploads/?mount=<digest>&from=<other_namespace>`  <sup>[end-11](#endpoints)</sup>.
```

Comment:
The original spec referred to this as `<repository name>` -- why change it?

There may have been other conversations about preferring namespace over repository that I missed. If so feel free to ignore me.

Reply:
no reason -- I reverted this back to `<repository_name>`